src/content/docs/en/recipes/docker.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Build your Astro Site with Docker
+description: How to build your Astro site to the web using Docker.
```

Comment:
Shouldn't this be "How to build & deploy your Astro site to the web using Docker"?

Reply:
Changed to "build your Astro site using Docker" since we don't teach how to deploy